tests/config/default.playwright.config.ts

```diff
@@ -71,7 +71,7 @@ if (mode === 'service') {
 
 if (mode === 'service2') {
   config.webServer = {
-    command: 'npx playwright run-server 3333',
```

Comment:
not sure if this breaks people, some are using this unofficial `run-server` command already.

Reply:
Well, they should not 😄